micro-rdk-installer/src/main.rs

```diff
@@ -167,27 +172,52 @@ fn flash(binary_path: &str, should_monitor: bool) -> Result<(), Error> {
     Ok(())
 }
 
+fn get_path_or_download(path: Option<String>, tmp_dir: &TempDir) -> Result<PathBuf, Error> {
+    Ok(match path {
+        Some(path) => PathBuf::from(path),
+        None => {
+            let rt = Runtime::new().map_err(Error::AsyncError)?;
```

Comment:
i think it's fine here to decorate the main function with the tokio macro

Reply:
I've refactored this a bit and not the run time is being used in main, is it still a good idea to use the macro?

Reply:
yes it would be better it's more inline with the rest of rust programs that uses tokio. In general unless you have a good reason for it you should avoid instantiating a runtime yourself